_includes/scripts.html

```diff
@@ -39,7 +39,13 @@
 });
 </script>
 
-<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB48gZNKOtEbslmLkTWtphkzCJVM6ITd1s&callback=initialize"></script>
+<script
+  src="https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/markerclusterer.js"
```

Comment:
The existence of `examples` in this URL path makes me think this is subject to change, and might not be future proof.   Thoughts, @stoe?